docs/source/tutorials/partition_table.rst

```diff
@@ -0,0 +1,188 @@
+###############
+Partition Table
+###############
+
+Introduction
+------------
+
+Partition table is used to define the flash memory organization and the different kind of data will be stored on each partition.
+
+You can use one of the predefined partition table schemes available or create your own.
```

Comment:
You can use one of the **available**(moved) predefined partition table schemes or create your own.